distribution.md

```diff
@@ -23,6 +24,56 @@ Distribution API versions:
 
 ## Artifact Format
 
+### Asset Package 
+
+An Asset Package MUST exist as and OCI image on a registry, and OCI image in a Docker daemon or as an uncompressed tar archive containing an OCI image.
+
+- For Linux asset packages, all FS layers MUST contain one or more assets.
```

Comment:
Do we define `FS` anywhere?

Reply:
We do not, but FS is used before this PR as well it seems.